picturae_csv_create.py

```diff
@@ -570,11 +563,22 @@ def col_clean(self):
         """parses and cleans dataframe columns until ready for upload.
             runs dependent function taxon concat
         """
-        # flagging taxons with multiple missing ranks
-        self.flag_missing_data()
+        # concatenate date
+
+        for col_name in list(["start", "end"]):
```

Comment:
moved these date formatting functions before the flag missing data function, as it makes the date QC checks easier.